src/main/java/core/basesyntax/service/parse/DataParserImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.parse;
+
+import core.basesyntax.model.FruitRecord;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataParserImpl implements DataParser {
+
+    @Override
+    public List<FruitRecord> parseFruitRecords(List<String> lines) {
+        List<FruitRecord> transactions = new ArrayList<>();
+
+        for (int i = 1; i < lines.size(); i++) {
```

Comment:
Avoid hardcoding the starting index of the loop. Instead, define a constant for the header line or dynamically determine if a line is a header to be skipped.